client/verta/tests/unit_tests/internal_utils/test_kafka_utils.py

```diff
@@ -2,81 +2,80 @@
 
 """Unit tests for Kafka utilities."""
 
-from typing import Any, Dict, List
+from typing import Any, Dict
 
-import pytest
+from hypothesis import given, HealthCheck, settings, strategies as st
 
 from verta._internal_utils import kafka
 
 
-@pytest.fixture
-def mock_kafka_configs_response() -> Dict[str, Any]:
+@st.composite
+def mock_kafka_configs_response(draw) -> Dict[str, Any]:
     """
     Provide mocked API result from `api/v1/uac-proxy/system_admin/listKafkaConfiguration`
     with a single Kafka configuration.
     """
+    recursive_json = st.recursive(
+        st.none() | st.booleans() | st.integers() | st.text(),
+        lambda children: st.lists(children) | st.dictionaries(st.text(), children),
+        max_leaves=5,
+    )
     return {
         "configurations": [
             {
-                "id": "1232abcdefg4-567h-891i-0jkl-1m1n1o2131pq",
-                "kerberos": {
-                    "enabled": True,
-                    "conf": (
-                        "kerberos.example.com"
-                    ),
-                    "keytab": "test-key-tab=",
-                    "client_name": "test-kafka-client_name",
-                    "service_name": "test-kafka-utils",
-                },
-                "brokerAddresses": "kerberos.example.com",
-                "enabled": True,
-                "name": "Test Kafka Name",
+                "id": draw(st.integers()),
+                "kerberos": draw(recursive_json),
```

Comment:
This doesn't seem right. `"kerberos"` isn't an arbitrarily nested unstructured JSON; it has specific keys and values, right?